store/job_storer.go

```diff
@@ -0,0 +1,43 @@
+package store
+
+import (
+	"context"
+	models "github.com/ONSdigital/dp-search-reindex-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"time"
+)
+
+type JobStorer struct {
+	JobsMap map[string]models.Job
+}
+
+// CreateJob creates a new Job resource and stores it in the JobsMap
+func (js *JobStorer) CreateJob(ctx context.Context, id string) (models.Job, error) {
```

Comment:
Do we need to validate the ID? I.e. check if its empty, is it a duplicate etc.

Reply:
yes, agreed as discussed - done